File: app/src/main/java/com/empcloud/empmonitor/data/remote/request/createleave/CreateLeaveRequestModel.kt
package com.empcloud.empmonitor.data.remote.request.createleave

import java.io.Serializable

data class CreateLeaveRequestModel(
    val day_type:Int,
    val leave_type:Int,
    val start_date:String,
    val end_date:String,
    val reason:String

    ):Serializable

File: iOS/EmpMonitor/EmpMonitor/Models/LeavesModels/CreateLeaveRequestModel.swift
//
//  CreateLeaveRequestModel.swift
//  EmpMonitor
//
//  Created by Sumit Ghosh on 14/08/24.
//

import Foundation

// MARK: - CreateLeaveRequestModel
struct CreateLeaveRequestModel: Codable {
    let dayType, leaveType: Int
    let startDate, endDate, reason: String

    enum CodingKeys: String, CodingKey {
        case dayType = "day_type"
        case leaveType = "leave_type"
        case startDate = "start_date"
        case endDate = "end_date"
        case reason
    }
}
